app/views/question_flow/cannot_use_service.html.slim

```diff
@@ -6,7 +6,8 @@
 .govuk-grid-column-two-thirds
   = render "shared/heading"
     h1.govuk-heading-xl = t("question_flow.property.cannot_use_service.title")
-    p.govuk-body = t("question_flow.property.cannot_use_service.paragraph_1_#{@check.level_of_help}#{'_partner' if @check.partner}")
+    p.govuk-body id='jointOwnersExitPage'
```

Comment:
ID creation in CCQ is generally (always?) automated by our dlim files and partials and they follow the convention of lowercase and hyphen separated, e.g:
 `id="other-income-form-maintenance-relevant-true-field"`
Could this ID be renamed to stick with this convention.

Reply:
@willc-work Have updated 🙂